src/ShaderCache.h

```diff
@@ -113,7 +113,8 @@ namespace SIE
 				       type == RE::BSShader::Type::Grass ||
 				       type == RE::BSShader::Type::Particle ||
 				       type == RE::BSShader::Type::Water ||
-				       type == RE::BSShader::Type::Effect;
+				       type == RE::BSShader::Type::Effect ||
+				       type == RE::BSShader::Type::Utility;
 			return type == RE::BSShader::Type::Lighting ||
```

Comment:
Saw some VR code in the `Utility.hlsl`, should we also add `type == RE::BSShader::Type::Utility` to the VR case here as well?

Reply:
No, I just added eyeIndex where it's required, but didn't do actual VR RE